package.json

```diff
@@ -53,7 +53,7 @@
   "dependencies": {
     "abort-controller": "^3.0.0",
     "bn.js": "^5.2.0",
-    "ethers": "^5.1.4",
+    "ethers": "5.1.4",
```

Comment:
Per `Experimental Feature` in https://docs.ethers.io/v5/api/utils/hashing/#TypedDataEncoder , I pinned this.